ClimateMonitorV2/views/our-team.pug

```diff
@@ -0,0 +1,162 @@
+extends layout
+
+block content
+    head
+    script(src="https://unpkg.com/leaflet@1.6.0/dist/leaflet.js", integrity="sha512-gZwIG9x3wUXg2hdXF6+rVkLF/0Vi9U8D2Ntg4Ga5I5BZpVkVxlJWbSQtXPSiUTtC0TjtGOmxa1AJPuV0CPthew==", crossorigin="")
+    script(src='https://api.mapbox.com/mapbox.js/plugins/leaflet-fullscreen/v1.0.1/Leaflet.fullscreen.min.js')
+    link(href='https://api.mapbox.com/mapbox.js/plugins/leaflet-fullscreen/v1.0.1/leaflet.fullscreen.css' rel='stylesheet')
+
+    link(rel='stylesheet' href='https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css')
+
+    script(src="https://cdnjs.cloudflare.com/ajax/libs/Chart.js/2.7.3/Chart.bundle.min.js")
+    script(src="https://cdn.jsdelivr.net/npm/hammerjs@2.0.8")
+    script(src="https://cdn.jsdelivr.net/npm/chartjs-plugin-zoom@0.7.7")
+
+    style
+        include stylesheets/style.css
+
+    body
+        div(class="sidebar")
+            div(class="sidebar-header")
+                a(href="https://github.com/dambem/ClimateMonitorV2", id="sheffsenseImageHolder", target="_blank")
+                    img(class="img-fluid", id="sheffsenseImage", src="sheffsense.png", alt="The Sheffsense logo: A colourful mosaic circle on a blurred landscape background")
+            div(class="sidebar-section")
+                img(class="sidebar-icon" src="icons/home.svg")
+                a(class="nav-link" href="/") Home
+            div(class="sidebar-section")
+                img(class="sidebar-icon" src="icons/info.svg")
+                a(class="nav-link" href="/about") About
+            div(class="sidebar-section")
+                img(class="sidebar-icon" src="icons/statistics.svg")
+                a(class="nav-link" href="/detailedstats") Statistics
+            div(class="sidebar-section-active")
+                img(class="sidebar-icon" src="icons/group.svg")
+                a(class="nav-link" href="/our-team") Our Team
+            div(class="sidebar-footer")
+                a(href="https://urbanflows.ac.uk/" target="_blank")
+                    img(class="img-fluid", id="urbanflowsimage", src="images/urbanlogo.png", alt="The Urban Flows logo: A ring of navy, lilac & green dots interlaced with a star made of purple dots")
+
+        div(id='the-team')
+        section.our-webcoderskull.padding-lg
+          .container
+            .row.heading.heading-icon
+              h2 Our Team
+            ul.row
+              li.col-12.col-md-6.col-lg-3
+                .cnt-block.equal-hight(style='height: 349px;')
+                  figure
+                    img.img-responsive(src='images/akshat.png' alt='')
+                  h3
+                    a Akshat Shah
+                  p Web Designer &amp; Documentation
+                  ul.follow-us.clearfix
+                    li
+                      a(href='https://github.com/au-shah')
```

Comment:
Can we add `target="_blank"` to each of our github and linkedin `a` tags? That way, they open in a new tab and it's a whole lot more nice doing that than having to click the back button several times 😉